Sources/Orbit/Support/Components/PasswordStrengthIndicator.swift

```diff
@@ -0,0 +1,118 @@
+import SwiftUI
+
+public enum PasswordStrength: Equatable {
+    case empty
+    case weak(title: String)
+    case medium(title: String)
+    case strong(title: String)
+}
+
+/// Orbit password strength indicator.
+public struct PasswordStrengthIndicator: View {
+
+    let passwordStrength: PasswordStrength
+    let textAccessibilityIdentifier: String
```

Comment:
Probably no need for that, as the caller can simply assign the ID through the only StaticText that is here.